ci/cirrus-templates/gen-workflows.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+INPATH=$(dirname "$0")
+OUTPATH="${INPATH}/../.."
+
+cp freebsd.yaml "$OUTPATH/.cirrus.yml"
```

Comment:
This whole file feels a bit odd to have, though I suppose we might want to template it in future.

Reply:
I kept the same logic as github actions. If it's useless, I can remove it. :)